pennylane/tape/qscript.py

```diff
@@ -835,13 +864,13 @@ def copy(self, copy_operations=False):
         )
         new_qscript._graph = None if copy_operations else self._graph
         new_qscript._specs = None
-        new_qscript.wires = copy.copy(self.wires)
-        new_qscript.num_wires = self.num_wires
-        new_qscript._update_par_info()
-        new_qscript._obs_sharing_wires = self._obs_sharing_wires
-        new_qscript._obs_sharing_wires_id = self._obs_sharing_wires_id
-        new_qscript._batch_size = self._batch_size
-        new_qscript._output_dim = self._output_dim
+        EXCLUDED_ATTR = ("_ops", "_measurements", "_measurements", "_shots", "_trainable_params")
```

Comment:
```suggestion
        EXCLUDED_ATTR = ("_ops", "_measurements", "_measurements", "_shots", "_trainable_params", "_specs", "_graph")
```
Since those are updated above as well.

Reply:
As discussed in slack, we decided to leave the old format to avoid complications.